api.yaml

```diff
@@ -52,44 +52,59 @@ paths:
       responses:
         "200":
           description: "Success."
+        "400":
```

Comment:
is this intended to be under the /healthcheck path?

Reply:
Nope, they're meant to be under the /notification path. Thanks